client/components/jetpack/card/jetpack-free-card/style.scss

```diff
@@ -29,24 +29,29 @@
 .jetpack-free-card__body {
 	padding: 24px 16px;
 
-	display: grid;
-	grid-template-rows: auto auto;
 	@include break-medium {
-		grid-template-columns: 50% 50%;
+		display: flex;
```

Comment:
Grid isn't well supported by IE. I believe we still want to support the latest version.